### ISSUE TITLE
Jodel bots

### ISSUE THREAD

**Issue by nikriek** [jafrewa/JodelPy#7]
Bots were my initial idea. How about creating some useful ones? E.g.:
 
- Post Mensa meals every day at 10 am using https://openmensa.org

**Comment by jafrewa** [comment]
I agree. The scripts need a -l/--location parameter first. And JodelREST
needs a Json file reader. Priorities young one.
Am 23.11.2015 6:13 nachm. schrieb "Niklas Riekenbrauck" <
notifications@github.com>:

**Comment by nikriek** [comment]
Ok master
